python.pri

```diff
@@ -0,0 +1,35 @@
+win32 {
+	PYTHON=python
+} else {
+	PYTHON=$$system(which python 2>/dev/null)
```

Comment:
Ah. Ok. Would be great to mention that directly in python.pri to prevent future confusion about it.